test/clj/matcher_combinators/core_test.clj

```diff
@@ -96,6 +83,8 @@
                     ::result/value (model/->Mismatch expected actual)}
                    res))))
 
+(spec.test/instrument)
```

Comment:
✂️

Reply:
It was there already at the top of the namespace, to ensure that the midje facts are run with instrumentation on. I think it should stay. WDYT now that you have the context?